src/test/kotlin/lotto/LottoSellerTest.kt

```diff
@@ -0,0 +1,14 @@
+package lotto
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import java.math.BigDecimal
+
+internal class LottoSellerTest {
+
+    @Test
+    fun `Lotto금액별 갯수`() {
```

Comment:
```suggestion
    fun `천 단위로 Lotto 개수를 얻는다`() {
```
Lotto금액별 갯수가 어떤 것을 테스트하는지 잘 모르겠네요 😭 
테스트 함수가 어떤 것을 테스트하는지 명시적으로 작성해보면 어떨까요?

Reply:
만약 천단위가 아닌 숫자(예, 1250)가 입력되면 어떻게 될까요?